src/main/scala/com/cognite/sdk/scala/v1/resources/sessions.scala

```diff
@@ -0,0 +1,102 @@
+// Copyright 2020 Cognite AS
+// SPDX-License-Identifier: Apache-2.0
+
+package com.cognite.sdk.scala.v1.resources
+
+import com.cognite.sdk.scala.common._
+import com.cognite.sdk.scala.v1._
+import io.circe.generic.semiauto.{deriveDecoder, deriveEncoder}
+import io.circe.{Decoder, Encoder, Printer}
+import sttp.client3._
+import sttp.client3.circe._
+
+class Sessions[F[_]](val requestSession: RequestSession[F])
+    extends WithRequestSession[F]
+    with BaseUrl {
+  import Sessions._
+  override val baseUrl = uri"${requestSession.baseUrl}/sessions"
+
+  def createWithTokenExchangeFlow(items: Items[SessionCreateWithToken]): F[Seq[Session]] = {
+    implicit val customPrinter: Printer = Printer.noSpaces.copy(dropNullValues = true)
```

Comment:
Seems all but get use this, perhaps we can lift it one level? Why do we need it btw?

Reply:
https://github.com/circe/circe/blob/master/modules/core/shared/src/main/scala/io/circe/Printer.scala#L32
My guess is that it is used to avoid printing null field in the Json, but we won't need them here so I just remove these implicit :)